### PR TITLE
Draft

### PR DESCRIPTION
**Pull request by Bala667**
Hi Norman

I am creating the Pull request for this git hub application example, As I made changes, Please review it

Thanks

### CODE REVIEW

File: docs/Installation.md [hunk 1]
```diff
@@ -21,15 +37,58 @@ Deploy the configuration.
...
 ## Configure OPC UA Connector
 
+OPC UA connector can be configured via both bellow mentioned options
+
+###### a) Local configuration with industrial edge device
+###### b) Central configuration with industrial edge managment
+
+#### Local Configuration
+
+In your industrial edge devic open the common congigurator and configure the OPC UA connector
```
Comment (HummelN7872) [inline docs/Installation.md:47]: spelling mistake "congigurator "

File: docs/Installation.md [hunk 2]
```diff
@@ -21,15 +37,58 @@ Deploy the configuration.
...
+OPC UA connector can be configured via both bellow mentioned options
+
+###### a) Local configuration with industrial edge device
+###### b) Central configuration with industrial edge managment
+
+#### Local Configuration
+
+In your industrial edge devic open the common congigurator and configure the OPC UA connector 
+
+This configuration done localy at the industrial edge Device
```
Comment (HummelN7872) [inline docs/Installation.md:49]: locally instead of localy

File: README.md [hunk 3]
```diff
@@ -66,15 +67,15 @@ You can find the further information about the following steps in the [docs](doc
 - Configure OPC UA Connector
 - Collect data in Flow Creator and calculate KPIs
 - Create custom data source (new metadata, publish data to new topic)
-- Install and configure OPC UA configurator and application
+- Install and configure OPC UA connector 
 
 ## Usage
 
-When previous steps are configured correctly, data are available in OPC UA Application. Use UA Expert to connect to OPC UA Application at end point `opc.tcp://Ip-Address-of-Edge-Device:48010`. You can also use second IED. Then data is obtained via OPC UA Connector.
+When previous steps are configured correctly, data are available in OPC UA Application. Use UA Expert to connect to OPC UA Application at end point `opc.tcp://Ip-Address-of-Edge-Device:4840`. You can also use second IED. Then data is obtained via OPC UA Connector.
```
Comment (HummelN7872) [inline README.md:74]: Are you sure about the port 4840? In the documentation it says that the correct port ist 48010

_A client can connect to Edge OPC UA server at end point opc.tcp://Ip-Address-ofEdge-Device:48010._

**Review by HummelN7872 — CHANGES_REQUESTED** [review]

**Comment by Bala667** [comment]
Hi Norman everything is updated you can merge it now, Thanks.

**Review by HummelN7872 — APPROVED** [review]

**Pull request merged**